src/styles/blocks/header.scss

```
@@ -0,0 +1,61 @@
+.logo {
+  margin-top: 4px;
+}
+
+.header {
+  font-weight: 500;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding-left: 50px;
+  padding-right: 50px;
+}
+
+.nav {
+  &__link {
+    position: relative;
+    line-height: 60px;
+    justify-content: center;
+    align-items: center;
+    text-align: left;
+    font-size: 12px;
+    color: $main-accent-color;
+    text-decoration: none;
+    text-transform: uppercase;
+  }
+
+  &__list {
+    display: flex;
+    list-style: none;
+    margin: 0;
+    padding: 0;
+  }
+
+  &__item {
+    display: flex;
+    margin-right: 20px;
+  }
+
+  &__item:last-child {
```

use scss nesting here
```suggestion
  &__item {
   ...
      &:last-child {
```